src/borg/repository.py

```diff
@@ -778,7 +778,6 @@ def complete_xfer(intermediate=True):
                              segment, freeable_ratio * 100.0, freeable_space)
                 pi.show()
                 continue
-            freed_space += freeable_space  # this is what we THINK we can free
```

Comment:
see comment, we just got that from hints and that says nothing about whether we really can free that amount.